### DISCUSSION TITLE
是否会提供 addGroup(添加群聊) API ？

### DISCUSSION THREAD

**Discussion by rikkihamano** [takayama-lily/oicq#404] [Q&A] [answered]
在如下文件中看到了： https://github.com/takayama-lily/oicq/blob/c33264b0cc7b2182dc152bbf3ac2694aac20618a/test/api.js#L54 但是 API 文档中却没有相关说明，实际也确实不存在这个 API     
    
找了几个地方，并没有找到相关讨论或资料  
   
对 js / ts 不是很了解，所以想请问一下各位

**Comment by takayama-lily** [comment, marked as answer]
不会提供

**Comment by dogxii** [comment]
接口风控
